Dockerfile_build

```diff
@@ -15,7 +15,7 @@ RUN apt-get update && \
     autoconf automake autotools-dev libtool xutils-dev valgrind && \
     rm -rf /var/lib/apt/lists/*
 
-# Download, verify, and install ragel version 6.
+# Download, verify, and install ragel version 6. 
 ENV RAGEL_VERSION=6.10
```

Comment:
Should this be changed to reference Ragel 7?

Reply:
Both versions are still in the Dockerfile since we're flagging ragel 7 and using 6 by default. I think this change came from a linter or a formatter or something.

Reply:
We're flagging the new ragel because we're flagging the scanner (I believe).

Reply:
I imagine we may remove these lines from the dockerfile once the new scanner is vetted in prod.

Reply:
and the flag removed of course.